doc/source/devices/index.rst

```diff
@@ -77,14 +77,15 @@ Supported Devices
    adi.cn0511
    adi.cn0532
    adi.cn0540
+   adi.cn0554
    adi.daq2
    adi.daq3
    adi.fmc_vna
    adi.fmcadc3
    adi.fmcjesdadc1
    adi.fmclidar1
-   adi.fmcomms5
    adi.fmcomms11
+   adi.fmcomms5
```

Comment:
FMComms5 should come before FMComms11

Reply:
rearranged fmcomms5 and fmcomms11 on index.rst file